main.tf

```diff
@@ -13,21 +13,21 @@ resource "aws_lambda_function" "default" {
   # The function_name, runtime, memory_size, and timeout use variables
   # to facilitate testing of both new runtimes and new function versions.
   # End users will ordinarily use the default values.
-  function_name = "${var.function_name}"
+  function_name = var.function_name
 
-  runtime     = "${var.runtime}"
-  memory_size = "${var.memory_size}"
-  timeout     = "${var.timeout}"
+  runtime     = var.runtime
+  memory_size = var.memory_size
+  timeout     = var.timeout
   handler     = "index.handler"
-  publish     = "true"
-  role        = "${aws_iam_role.default.arn}"
+  publish     = true
+  role        = aws_iam_role.default.arn
   s3_bucket   = "drone-${local.region}-${local.account_id}"
   s3_key      = "splunk-aws-serverless-apps/splunk-cloudwatch-logs-processor.zip"
```

Comment:
We need to do something about this hardcoded s3 key.

Reply:
Again, this should be reviewed. This module was created  to deploy a single lambda function, but ought to be made more generic if possible. The S3 key being hardcoded would preclude this. I've added this to #3.

Reply:
Let's do this in the same card where we refactor `terraform-aws-cloudfront-directory-index`. They are both related, and if we can show that they work cleanly, I am more than happy to eliminate the S3 bucket once it becomes unnecessary.